packages/twig/src/components/nav/desktop/navsearch.twig

```diff
@@ -0,0 +1,25 @@
+{# navsearch.twig #}
+{% set prefix = prefix|default("ilo") %}
+{% set base_class = prefix ~ "--nav-dropdown" %}
+
+<template id="{{ prefix }}--nav-search__template">
+	<div class="{{ base_class }} {{ className|default("") }}" id="{{ base_class }}" tabindex="0">
+		<div class="{{ base_class }}__container">
+			{% include '@components/form/form.twig' with {
+      attributes: {
```

Comment:
I think it is better if the variables of the include are indented some more so that the closing bracket `%}` is at the same level as the opening one. 

Ideally these kind of things should be tackled by a linter and lint rules but I'm not sure whether we have one set up for the twig project.